attributes/default.rb

```diff
@@ -39,6 +47,9 @@
 when 'openbsd', 'freebsd', 'mac_os_x'
   default['ssl_certificate']['key_dir'] = '/etc/ssl'
   default['ssl_certificate']['cert_dir'] = '/etc/ssl'
+when 'windows'
+  default['ssl_certificate']['key_dir'] = Chef::Config[:file_cache_path]
+  default['ssl_certificate']['cert_dir'] = Chef::Config[:file_cache_path]
```

Comment:
You could use any directory of your choice, but as you may know Windows is not into storing system data in simple file, it would rather use the registry, etc.
There is no specific folder for storing certificate, except `%SystemRoot%\System32\Certlog` when you have your own CA - but in this case you don't need this cookbook.
I think the most common use case on windows, is to generate the certificate then install it in the right store - using `windows_certificate` LWRP for instance, so cache folder was good for me.

So ... as you wish, `file_cache_path` or `System32\Certlog` ?